src/api/java/baritone/api/Settings.java

```diff
@@ -1183,6 +1183,21 @@
      */
     public final Setting<Boolean> notificationOnMineFail = new Setting<>(true);
 
+    /**
+     * Disallow block placement and breaking within a range from the home waypoint(if it exists)
+     */
+    public final Setting<Double> homeProtectionRange = new Setting<>(0.0);
```

Comment:
Why not an Integer? I don't think people will need sub-block precision for their home protections and ints are easier to deal with when checking whether Baritone is inside that range.

Reply:
Yeah a double would prob be a bit overkill for such a task

Reply:
Changed in last commit